src/lib/circuit_value.ts

```diff
@@ -5,29 +5,36 @@ type Constructor<T> = { new (...args: any[]): T };
 
 export type Tuple<A, _N extends number> = Array<A>;
```

Comment:
This `Tuple` type isn't used anywhere, I'd generally prefer to remove exports that are neither used nor documented